trunner/runners/wrappers.py

```diff
@@ -0,0 +1,335 @@
+#
+# Phoenix-RTOS test runner
+#
+# Wrappers for programs used by Test Runner
+#
+# Copyright 2021, 2022 Phoenix Systems
+# Authors: Jakub Sarzyński, Mateusz Niewiadomski, Damian Loewnau
+#
+
+import logging
+import os
+import pexpect
+import signal
+import threading
+
+from abc import ABC, abstractmethod
+
+from trunner.tools.color import Color
+from .common import boot_dir, wait_for_dev
+
+
+def is_github_actions():
+    return os.getenv('GITHUB_ACTIONS', False)
+
+
+def append_output(progname, message, output):
+    progname = progname.upper()
+
+    msg = message
+    msg += Color.colorify(f'\n{progname} OUTPUT:\n', Color.BOLD)
+    msg += f'------------------------\n{output}\n------------------------\n'
+
+    return msg
+
+
+class PsuError(Exception):
+    def __init__(self, error):
+
+        msg = f'{Color.BOLD}PSU ERROR:{Color.END} {error}\n'
+        super().__init__(msg)
+
+
+class GdbError(Exception):
+    def __init__(self, error, gdb_output='', gdbserv_output=''):
+
+        msg = f'{Color.BOLD}GDB ERROR:{Color.END} {error}\n'
+        if gdb_output:
+            msg = append_output('gdb', msg, gdb_output)
+        if gdbserv_output:
+            msg = append_output('JlinkGdbServer', msg, gdbserv_output)
+
+        super().__init__(msg)
+
+
+class PhoenixdError(Exception):
+    def __init__(self, message, output=''):
+
+        msg = f'{Color.BOLD}PHOENIXD ERROR:{Color.END} {message}\n'
+        if output:
+            msg += f'{Color.BOLD}PHOENIXD OUTPUT:{Color.END}\n {output}\n'
+
+        super().__init__(msg)
+
+
+class StandardWrapper(ABC):
```

Comment:
I don't know if the wrapper is not too generic for the file name and classes. It wraps the way how we handle the process.

Reply:
I've changed the naming in this file. The whole file is called `utils.py` to be consistent with the naming in `phoenix-rtos-project` and wrappers are now called ProcessHandlers